src/nodedef.h

```diff
@@ -422,4 +427,3 @@ class NodeResolver {
 
 
 #endif
-
```

Comment:
GitHub is screwing with you, there were two newlines there, now there's only one.

If there are no new lines, GH usually displaces a red symbol with a tooltip of "no trailing newline" or something.